module3/homework/vectorOfSharedPointers/vectorFunctions.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+std::vector<std::shared_ptr<int>> generate(int);
```

Comment:
All other function declarations have parameters given as `(type name)` and here you have only `(type)` - it's not a mistake in terms of compilation, but to have uniform approach in whole file it would be nice to add a name of parameter. For example `int count` as it is in function definition in .cpp file.